### DISCUSSION TITLE
Typography "code" broken

### DISCUSSION THREAD

**Discussion by kenmorse** [tailwindlabs/tailwindcss#9192] [Help] [answered]
Using the Typography plugin, I see backticks are output with code — is that correct behavior?

Here's a simple example:

        <div class="prose">
          <h1>Using Prose</h1>
          <p>And some text so we know Typography is hooked up</p>
          <p>And now some <code>inline stuff</code> in the mix</p>
        </div>

The output will show backticks around "inline stuff".

Can reproduce even with the Play CDN.

**Comment by wongjn** [comment, marked as answer]
Yup, they are there by design I would say – [the backticks are declared in the styles from the typography plugin](https://github.com/tailwindlabs/tailwindcss-typography/blob/c1f01be2e6c7b13d03bee7789ec0378ba1dfca63/src/styles.js#L1361-L1366):

```js
module.exports = {
  DEFAULT: {
    css: [
      {
        // …
        'code::before': {
          content: '"`"',
        },
        'code::after': {
          content: '"`"',
        },
        // …
      },
    ],
  },
  // …
};
```

**Reply by kenmorse** [reply]
@wongjn — yes, I noticed that too. I was just surprised this was the default coming at this from a markdown POV — in that case for inline code you use backticks for the input, but the output (including styling) typically doesn't contain them. GitHub itself converts markdown and the output does not include the backticks. No big deal as the typography plugin is so configurable.

**Reply by adamwathan** [reply]
Yeah just a stylistic choice, we thought it looked cool when we saw it on the Vercel blog. We tried a solid background color originally but just didn’t feel like it looked great.
